runtime/onert/core/src/compiler/OperationValidator.cc

```diff
@@ -1292,5 +1292,17 @@ void OperationValidator::visit(const ir::operation::LogicalOr &node)
   OP_REQUIRES(_ctx.at(lhs_index).typeInfo().type() == _ctx.at(output_index).typeInfo().type());
 }
 
+void OperationValidator::visit(const ir::operation::Range &node)
+{
+  const auto output_index{node.getOutputs().at(0)};
+  const auto start_index{node.getInputs().at(ir::operation::Range::Input::START)};
+  const auto limit_index{node.getInputs().at(ir::operation::Range::Input::LIMIT)};
+  const auto delta_index{node.getInputs().at(ir::operation::Range::Input::DELTA)};
+
+  UNUSED_RELEASE(output_index);
+  UNUSED_RELEASE(start_index);
+  UNUSED_RELEASE(limit_index);
+  UNUSED_RELEASE(delta_index);
```

Comment:
@intom There are no validation code at all. Do you write this code in advance for someone who may add actual validation?

Reply:
I will remove and add the dimension check code.